src/sync/mod.rs

```diff
@@ -1 +1,4 @@
+//! Synchronization primitives.
+
 pub mod list;
```

Comment:
Do we want to expose this `List` implementation to the outside world? Imo we should keep it as a detail of the epoch GC since e.g. the `Iter` requires `Scope`s, but as I understand scopes are only used when implementing data structures. When using them, there should be a simpler API.